### ISSUE TITLE
flake-stats: create overview page to ease browsing of history

### ISSUE THREAD

**Issue by dhiller** [kubevirt/project-infra#2935]
# TL;DR
goal: write a go tool that generates an overview page for the flake-stats reports and add it to the set of go commands located in [`robots/cmd`](https://github.com/kubevirt/project-infra/tree/main/robots/cmd) folder in [`kubevirt/project-infra`](https://github.com/kubevirt/project-infra) repository.

# Task: create a flake-stats-overview.html

We use the [flake-stats](https://github.com/kubevirt/project-infra/tree/main/robots/cmd/flake-stats) tool to generate an overview of how we are doing wrt flakes in [k/kubevirt](https://github.com/kubevirt/kubevirt)

This tool generates a daily overview [here](https://storage.googleapis.com/kubevirt-prow/reports/flakefinder/kubevirt/kubevirt/flake-stats-14days.html).

We want to generate a landing page that links directly to each of the reports for the last, say 90 days. Or even more, we wouldn't really care whether the actual report exists.

This i.e. would integrate the individual flake-stats reports into a new `flake-stats-overview.html`. For easier browsing this should group the entries and display them newest first.

Grouping could be done monthly or week-wise.

Extra sugar would be some sort of logical arrangement that helps the user to find the report he wants quickly. I.e. a calendar view. 

# Background

All flake-stats pages are accessible from google cloud storage and browseable via http:

```
$ gsutil ls gs://kubevirt-prow/reports/flakefinder/kubevirt/kubevirt/ | grep flake-stats
gs://kubevirt-prow/reports/flakefinder/kubevirt/kubevirt/flake-stats-14days-2023-07-19.html
gs://kubevirt-prow/reports/flakefinder/kubevirt/kubevirt/flake-stats-14days-2023-07-20.html
gs://kubevirt-prow/reports/flakefinder/kubevirt/kubevirt/flake-stats-14days-2023-07-21.html
...
gs://kubevirt-prow/reports/flakefinder/kubevirt/kubevirt/flake-stats-14days-2023-08-13.html
gs://kubevirt-prow/reports/flakefinder/kubevirt/kubevirt/flake-stats-14days-2023-08-14.html
gs://kubevirt-prow/reports/flakefinder/kubevirt/kubevirt/flake-stats-14days-2023-08-15.html
gs://kubevirt-prow/reports/flakefinder/kubevirt/kubevirt/flake-stats-14days.html
```

**Note: last line shows the latest report**

**Comment by anishbista60** [comment]
@dhiller can i work here ?

**Comment by dhiller** [comment]
> @dhiller can i work here ?

Sure, just go ahead.
